include/ada/url_aggregator-inl.h

```diff
@@ -51,17 +50,17 @@ inline void url_aggregator::update_base_authority(
     std::string_view password = input.substr(password_delimiter + 1);
 
     buffer.insert(components.protocol_end + diff, username);
-    diff += uint32_t(username.size());
+    diff += static_cast<uint32_t>(username.size());
```

Comment:
Why would

```C++
diff += uint32_t(username.size());
```
be replaced by...

```
diff += static_cast<uint32_t>(username.size());
```

When is the latter more correct?

Reply:
c-style cast looks better and more readable, I give you that, but it doesn't show the intent, and it certainly doesn't show which type of cast we need, and also it's trivially not searchable.